### PR TITLE
Re-up to NDK r25, add unwind flag to arm64

### PR DESCRIPTION
**Pull request by nirvn**

### CODE REVIEW

File: .github/workflows/android.yml [hunk 1]
```diff
@@ -36,9 +36,9 @@ jobs:
         id: vars
         shell: bash
         run: |
-          echo "ANDROID_NDK_VERSION=22.1.7171670" >> $GITHUB_ENV
+          echo "ANDROID_NDK_VERSION=25.1.8937393" >> $GITHUB_ENV
```
Comment (m-kuhn) [inline .github/workflows/android.yml:39]:
```suggestion
          ANDROID_NDK_VERSION=25.1.8937393
          echo "ANDROID_NDK_VERSION=${ANDROID_NDK_VERSION}" >> $GITHUB_ENV
          echo "ANDROID_NDK_HOME=/usr/local/lib/android/sdk/ndk/${ANDROID_NDK_VERSION}" >> $GITHUB_ENV
```

File: .github/workflows/android.yml [hunk 2]
```diff
@@ -36,9 +36,9 @@ jobs:
...
           echo "ANDROID_BUILD_TOOLS_VERSION=29.0.2" >> $GITHUB_ENV
-          echo "ANDROID_NDK_HOME=/usr/local/lib/android/sdk/ndk/22.1.7171670" >> $GITHUB_ENV
+          echo "ANDROID_NDK_HOME=/usr/local/lib/android/sdk/ndk/25.1.8937393" >> $GITHUB_ENV
```
Comment (m-kuhn) [inline .github/workflows/android.yml:41]:
```suggestion
```

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 4b3f65f752aa78f4dabaedcfe74c1dcd1c9b64d7: [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3349-ae5a6c14bc09d79cedc19c2981e8d29e8b4ac569-arm64-android.apk)
<details>
<summary>Other architectures</summary>

- [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3349-ae5a6c14bc09d79cedc19c2981e8d29e8b4ac569-arm-android.apk)
- [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3349-ae5a6c14bc09d79cedc19c2981e8d29e8b4ac569-x64-android.apk)
- [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3349-ae5a6c14bc09d79cedc19c2981e8d29e8b4ac569-x86-android.apk)
</details>

File: .github/workflows/android.yml (hunk 1 above)
Comment (m-kuhn) [reply .github/workflows/android.yml:39]: @nirvn did this get lost rebasing or was it broken?

Comment (nirvn) [reply .github/workflows/android.yml]: Lost when I pushed something locally and overwrote your change. But, it also turns out to not work for a reason I can't explain :/

**Pull request closed without merging**